.eslintrc.js

```diff
@@ -63,6 +63,7 @@ module.exports = {
     'newline-per-chained-call': 0,
     'no-confusing-arrow': 0,
     'no-console': 1,
+    'no-nested-ternary': 0,
```

Comment:
This will lead to unreadable code constructs. I'm not in favor of introducing this.
@aditudorach, @jpoppe; your thoughts?

Reply:
I agree with JJ. Le't keep this out.

Reply:
It added it because of this line: https://github.com/Amsterdam/signals-frontend/pull/985/files#diff-a1eeb871238ab0872eae54894bc0d3e4R26
I see your point, but it hurts me if I would have to write that differently.

Reply:
I also **strongly** agree we should avoid nested ternary expressions (JOKE: we are writing Javascript and not Perl :laughing: )

Reply:
OK. I will take it out!